packages/exoflex/src/components/Button/ButtonOpacity.tsx

```diff
@@ -35,47 +37,83 @@ export default function ButtonOpacity(props: ButtonProps) {
   });
   let { style: themeStyle } = useTheme();
 
+  const animatedOpacity = useRef(new Animated.Value(1)).current;
+  const fadeOpacityIn = useCallback(() => {
+    Animated.timing(animatedOpacity, {
+      toValue: preset === 'primary' ? 0.8 : 0.5,
+      duration: 100,
+      useNativeDriver: true,
+    }).start();
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, []);
```

Comment:
Let's not disable this lint rule and fix the warning.

Not setting the dependency for useCallback correctly might cause subtle bugs with stale values.

Actually, let's do optimization separately now that I think about it. See my next comment.